client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/index.tsx

```diff
@@ -1,185 +1,129 @@
-									<Button
-										primary
-										compact
-										className="site-migration-instructions__get-key-button component-button"
-										onClick={ getMigrationKey }
-									>
-										{ translate( 'Get migration key', {
-											components: {
-												strong: <strong />,
-											},
-										} ) }
-									</Button>
-								) }
-								{ siteMigrationKeyError && (
-									<>
-										{ translate(
-											'We were unable to retrieve your migration key. To get the key manually, go to the {{a}}Migrate Guru page on the new WordPress.com site{{/a}}.',
-											{
-												components: {
-													a: <a href={ destSiteUrl } target="_blank" rel="noreferrer" />,
-												},
-											}
-										) }
-									</>
-								) }
-							</>
-						) }
-						{ siteMigrationKey && ! siteMigrationKeyError && (
-							<>
-								{ translate(
-									'Click {{strong}}Copy key{{/strong}} below to copy your migration key - you will need that in a few minutes to start the migration. This key is unique to your site and will only be available once.',
-									{
-										components: {
-											strong: <strong />,
-										},
-									}
-								) }
-								<div className="site-migration-instructions__migration-key">
-									<code className="site-migration-instructions__key">
-										{ hideSiteMigrationKey
-											? '*'.repeat( siteMigrationKey.length - 5 )
-											: siteMigrationKey }
-									</code>
-									<Button
-										onClick={ () => {
-											setHideSiteMigrationKey( ! hideSiteMigrationKey );
-										} }
-										className="site-migration-instructions__show-key-button components-button"
-									>
-										{ hideSiteMigrationKey ? translate( 'Show key' ) : translate( 'Hide key' ) }
-									</Button>
-									<ClipboardButton
-										text={ siteMigrationKey }
-										className="site-migration-instructions__copy-key-button is-secondary"
-										onCopy={ onCopy }
-									>
-										{ buttonTextCopy ? translate( 'Copied!' ) : translate( 'Copy key' ) }
-									</ClipboardButton>
-								</div>
-							</>
-						) }
-					</div>
-				</li>
 				<li>
 					{ translate(
-						'Go to the {{a}}Migrate Guru page on the source site{{/a}}, enter your email address, and click {{strong}}Migrate{{/strong}}.',
+						'Install and active the {{a}}Migrate Guru plugin{{/a}} on your existing site.',
 						{
 							components: {
-								a: <a href={ sourceSiteUrl } target="_blank" rel="noreferrer" />,
-								strong: <strong />,
+								a: <a href={ getInstallationURL( fromUrl ) } target="_blank" rel="noreferrer" />,
 							},
 						}
 					) }
 				</li>
 				<li>
 					{ translate(
-						'You will see a screen showing multiple hosting providers. Select {{em}}Automattic{{/em}} as the destination host.',
+						'Go to the {{a}}Migrate Guru page on the source site{{/a}}, enter your email address, and click {{strong}}Migrate{{/strong}}.',
```

Comment:
Minor note (which is missing _all_ the context!): might it be clearer if we used "on ~the~ your source site"?

Reply:
Fixed here https://github.com/Automattic/wp-calypso/pull/88813/commits/658ae3d6c4e12a09c2d425af37902f8c47574824